cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -32,22 +32,54 @@
   not_if "hdfs dfs -test -d /var/log/hadoop-yarn", :user => "hdfs"
 end
 
+bash "create-hdfs-yarn-ats-log" do
+  code <<-EOH
+  hdfs dfs -mkdir -p /var/log/ats && hdfs dfs -chmod 1777 /var/log/ats && hdfs dfs -chown yarn:mapred /var/log/ats
```

Comment:
The current version of webhdfs does not support kerberos, neither does the resource in question, it would be relatively trivial to add all of that, but would like to leave that and the testing of thereof for later releases